great_expectations/datasource/fluent/sql_datasource.py

```diff
@@ -11,6 +11,7 @@
     Literal,
     Optional,
     Protocol,
+    Tuple,
```

Comment:
Nit - don't need to import this we can just use `tuple`

Reply:
Fix committed -- thanks.